mmocr/models/ner/loss/masked_focal_loss.py

```diff
@@ -0,0 +1,41 @@
+from torch import nn
+
+from mmdet.models.builder import LOSSES
+from mmocr.models.common.losses.focal_loss import FocalLoss
+
+
+@LOSSES.register_module()
+class MaskedFocalLoss(nn.Module):
+    """The implementation of masked focal loss."""
```

Comment:
Give a description on

* What is a masked focal loss, and
* Where the mask is defined

Reply:
The current description does not give any useful info